app/src/main/java/com/example/coinage/ParseApplication.java

```diff
@@ -0,0 +1,20 @@
+package com.example.coinage;
+
+import android.app.Application;
+
+import com.parse.Parse;
+import com.parse.ParseObject;
+
+public class ParseApplication extends Application {
+    // initializes parse sdk as soon as the application is created
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        Parse.initialize(new Parse.Configuration.Builder(this)
+                .applicationId("DZcZ5t4E5OwIQLRyPNYvprBQyD1G0Oy5nQlqrtpz")
```

Comment:
shall we move the hard coded values into constants! google android/java constants coding convention :-)